mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxACL.java

```diff
@@ -19,6 +19,7 @@
 
```

Comment:
commit name?

Reply:
?

Reply:
you have a " Continue MailboxACL refactoring " commit

Reply:
Changed to a fixup of the "interface" stuff.